pkg/version/version.go

```diff
@@ -16,4 +16,4 @@
 package version
 
 // BundlectlVersion is the version used by bundlectl.
-const BundlectlVersion = "0.12.0"
+const BundlectlVersion = "0.12.3"
```

Comment:
Is there a reason this is being bumped to .3 instead of .1?

Reply:
Uhh. because I forgot to do it the previous 2 releases